guideline/cfh.md

```diff
@@ -131,26 +131,11 @@ off by the PAC before they can be put in place.
    Proposals will have to provide details about how these can be implemented at
    the proposed venues.
 
-   1. Food and drinks for lunches and breaks on all conference days (and
```

Comment:
If we think lunches is a must-have for PyCon APAC, we should require host proposers to state how they plan to provide it.  @darjeeling gave a good example justifying that requirement.  If for any reason there is a difficulty to provide lunches, PAC should know it by reading the proposal.  It's PAC's duty to include this factor into their evaluation.